apiserver/facades/client/cloud/instance_information_test.go

```diff
@@ -101,22 +122,15 @@ func (p *instanceTypesSuite) TestInstanceTypes(c *gc.C) {
 	c.Assert(r.Results, gc.DeepEquals, expected)
 }
 
-func (*mockBackend) ModelConfig() (*config.Config, error) {
-	return nil, nil
-}
-
-func (b *mockBackend) CloudCredential(tag names.CloudCredentialTag) (state.Credential, error) {
-	return state.Credential{}, nil
-}
-
 type mockEnviron struct {
```

Comment:
Would it be feasible to drop this for a gomock generated MockEnviron? It's only used in the one test above, and doesn't look like anything complex is done with it